client/src/DisplayGrids/index.ts

```diff
@@ -0,0 +1,3 @@
+export { BulletGrid } from './BulletGrid';
+export { InstructionGrid } from './InstructionGrid';
```

Comment:
in `InstructionGrid` you can do `export default InstructionGrid;`
and here you could do `export { default as InstructionGrid } from './InstructionGrid';`